client/app/billing/autoRenew/autorenew.routing.js

```diff
@@ -87,7 +89,15 @@ export default /* @ngInject */ ($stateProvider) => {
         .serviceHasAutomaticRenew(service),
 
       nicBilling: /* @ngInject */ $transition$ => $transition$.params().nicBilling,
-      nicRenew: /* @ngInject */ BillingAutoRenew => BillingAutoRenew.getNicRenew(),
+      nicRenew: /* @ngInject */ (
+        BillingAutoRenew,
+        services,
+      ) => BillingAutoRenew.getAutorenew()
+        .then(nicRenew => ({
+          ...nicRenew,
+          isMandatory: services.userMustApproveAutoRenew,
+          renewDays: _.range(1, 30),
```

Comment:
No `31` ? `30` even for February ?

Reply:
ISO production, API only requires a renewDay so they handle February I guess

Reply:
Could you please add a comment next to it then ? Because it could easily be mistaken for an oversight :)